src/main/java/racingcar/Application.java

```diff
@@ -1,7 +1,21 @@
 package racingcar;
 
+import java.util.Scanner;
+import java.io.IOException;
+import java.util.Random;
+
 public class Application {
-    public static void main(String[] args) {
-        // TODO: 프로그램 구현
+    public static <string, try1> void main(String[] args) {
+        Scanner in = null;
+        System.out.println("자동차 이름을 적으시오 : ");
+        string cars = (string) in.next();
+        System.out.println("자동차 이름을 적으시오 : ");
+        string cars2 = (string) in.next();
+        System.out.println("자동차 이름을 적으시오 : ");
+        string cars3 = (string) in.next();
+
+        System.out.println("이동 횟수를 적으시오 : ");
+        try1 try1 = (try1) in.next();
```

Comment:
try1이라는 변수를 통해서 "정수"를 받고 싶으셨던 거 같아요.
Int를 받고 싶으시다면 아래처럼 선언해야해요! 이 관련된 부분도 위의 스캐너 블로그를 참고하시면 알기 좋을 거 같아요.
```suggestion
        int tryCount = in.nextInt(); // 
```